### PR TITLE
docs: update sdk documentation in readme

### PR DESCRIPTION
**Pull request by kevinpagtakhan**
### Summary

* Updates `README.md` in project root to specify V1 and V2 and their respective links
* Updates `CONTRIBUTING.md` to include instructions to fork, which branch to branch off and create a pull request against.
* Updates Browser SDK 2.0 `README.md` to only include quick start examples and refer to developer center for in-depth documentation.

### Checklist

* [x] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-TypeScript/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:  No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,16 +18,18 @@ If you find any existing issues that you are interested in fixing, you are welco
 
 Getting setup is quick and easy. Follow the steps below to get your your dev environment up.
 
-1. Clone GitHub repo
+1. Fork GitHub repo
```
Comment (kevinpagtakhan) [inline CONTRIBUTING.md:21]: notifying @kwalker3690

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,16 +18,18 @@ If you find any existing issues that you are interested in fixing, you are welco
...
+3. Build packages
 
 ```
-$ git clone git@github.com:amplitude/Amplitude-TypeScript.git
+$ git clone <HTTPS_OR_GIT_URL>
 $ yarn
 $ yarn build
 ```
 
+This repo supports mutliple major versions of all packages. For contributions to version `1.x`, create a branch off `v1.x`. For contributions to the `2.x` (latest) version, create a branch off `main`.
```
Comment (kevinpagtakhan) [inline CONTRIBUTING.md:31]: notifying @yuhao900914

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,6 +45,8 @@ When writing commit message, follow [PR Commit Title Conventions](#PR-Commit-Tit
 
 Once you are finished with your changes and feel good about the proposed changes, create a pull request. A team member will assist in getting them reviewed. We are excited to work with you on this.
 
+For contributions to version `1.x`, open a pull request against `v1.x`. For contributions to the `2.x` (latest) version, open a pull request against `main`.
```
Comment (kevinpagtakhan) [inline CONTRIBUTING.md:48]: notifying @yuhao900914

Comment (yuhao900914) [reply CONTRIBUTING.md:48]: Can we explain more details here about the situation for different packages? 
Like public node and rn and ma are in v1.x?

Comment (kevinpagtakhan) [reply CONTRIBUTING.md:61]: done

**Review by justin-fiedler — APPROVED** [review]

**Review by yuhao900914 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,16 +18,31 @@ If you find any existing issues that you are interested in fixing, you are welco
...
 ```
-$ git clone git@github.com:amplitude/Amplitude-TypeScript.git
+$ git clone <HTTPS_OR_GIT_URL>
 $ yarn
 $ yarn build
 ```
 
+This repo contains mutliple major versions of all packages. For contributions to version `1.x`, create a branch off `v1.x`. For contributions to the `2.x` (latest) version, create a branch off `main`. Refer to the table below for more infomation about the release status of each package.
+
+|Package|Version|Status|Dev Branch
```
Comment (yuhao900914) [inline CONTRIBUTING.md:33]: Missing an end `|`?

Comment (kevinpagtakhan) [reply CONTRIBUTING.md:33]: i think it still works without but i can add

Comment (kevinpagtakhan) [reply CONTRIBUTING.md:33]:
```suggestion
|Package|Version|Status|Dev Branch|
```

Comment (yuhao900914) [reply CONTRIBUTING.md]: Thanks.

**Pull request merged**